less/index.less

```diff
@@ -54,6 +54,271 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
 }
 
+html {
+	font-size: 62.5%;
+}
+
+body {
+	background-color: lightgray;
+}
+
+.container {
+	background-color: @main-color;
+	max-width: 800px;
+	margin: 0 auto;
+	font-family: 'Mukta', sans-serif;
+	
+		nav {
+			background-color: @first-accent;
+			width: 100%;
+			height: 50px;
+			.flexbox(row, nowrap);
+			.flexalign(flex-end, center, center);
+			padding-right: 10px;
+
+				@media (max-width: 500px) {
+					.flexalign(space-around, center, center);
+					padding-right: 0;
+				}
+				
+				a {
+					.plainlink();
+					font-size: 2rem;
+					padding-left: 20px;
+					font-family: @header-font;
+
+					@media (max-width: 500px) {
+						padding-left: 0;
+					}
+				}
+
+				a:hover {
+					color: black;
+				}
+		}
+
+		.top {
+			background-color: @base-color;
+			padding: 0 0 30px 20px;
+
+				img {
+					.dropshadow();
+					margin-top: 20px;
+
+					@media (max-width: 500px) {
+						width: 60%;
+						margin: 10px 20%;
+					}
+				}
+
+				h1 {
+					font-size: 4.5rem;
+					margin-top: 20px;
+					font-family: @header-font;
+					
+					@media (max-width: 500px) {
+						text-align: center;
+					}
+				}
+
+				h2 {
+					font-size: 2.7rem;
+					margin-top: 10px;
+					font-family: @header-font;
+
+					@media (max-width: 500px) {
+						text-align: center;
+					}
+				}
+
+				hr {
+					width: 90%;
+					margin: 15px 5%;
+				}
+
+				p {
+					font-size: 1.6rem;
+					padding-bottom: 20px;
+				}
+
+				.contact-button {
+					width: 150px;
+					height: 36px;
+					background-color: @second-accent;
+					border: 1px solid black;
+					cursor: pointer;
+					border-radius: 10px;
+					
+						p {
+							padding-top: 10px;
+							font-size: 1.6rem;
+							padding-left: 35px;
+						}
+				}
+		}
+
+		.about {
+			background-color: @main-color;
+			.flexbox(row, nowrap);
+			width: 100%;
+			padding: 20px 0 20px 4%;
+
+				@media (max-width:500px){
```

Comment:
Your nesting and use of mixins & variables looks great. You could add a variable for max-width:500px called @phone or @mobile then you can just write @media @phone. This is useful in case you want to change the px breakpoint without having to manually update each of the media queries :)